src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -197,13 +197,22 @@
               <toolTipText value="On save, hot reload changes into running Flutter apps."/>
             </properties>
           </component>
-          <component id="6d6f0" class="javax.swing.JCheckBox" binding="myTrackWidgetCreationCheckBox">
+          <component id="6d6f0" class="javax.swing.JCheckBox" binding="myDisableTrackWidgetCreationCheckBox">
             <constraints>
               <grid row="2" column="0" row-span="1" col-span="2" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
             </constraints>
             <properties>
-              <text value="Run applications with --track-widget-creation"/>
-              <toolTipText value="Allows the Flutter Inspector to track source locations where widgets are created. This setting is required for advanced Flutter Inspector functionality."/>
+              <text value="Disable tracking widget creation locations"/>
+              <toolTipText value="Disabling this setting breaks advanced Flutter Inspector functionality and performance tools. Only disable if you are noticing unexpected behavior differences between apps run in debug and release build."/>
+            </properties>
+          </component>
+          <component id="6d6f0" class="javax.swing.JCheckBox" binding="myLegacyTrackWidgetCreationCheckBox">
+            <constraints>
+              <grid row="3" column="0" row-span="1" col-span="2" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="[Flutter builds before 0.10.2 only] Run applications with --track-widget-creation"/>
```

Comment:
Instead of `[Flutter builds before 0.10.2 only] Run applications with --track-widget-creation`, perhaps: `Run applications with --track-widget-creation (always enabled for Flutter v0.10.2 and later)`?

Reply:
Sounds good. Done.